### PR TITLE
feat(standups): Standups responses grid UI improvements

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

Partially Fixes `Standups: Improve mobile experience #6601`

## Demo

https://user-images.githubusercontent.com/1017620/172626445-1b06248d-7960-429d-808e-64bf31f9d508.mp4

## Testing scenarios

[Please list all the testing scenarios a reviewer has to check before approving the PR]

- [ ] Open chrome responsive design mode and check all the most popular devices
- [ ] Open safari responsive design mode and check all the most popular devices
- [ ] Open firefox responsive design mode and check all the most popular devices

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx [hunk 1]
```diff
@@ -38,8 +37,19 @@ const ResponseWrapper = styled('div')<{
...
+    width: `calc(100% / 2 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [threeColumnResponseMediaQuery]: {
+    width: `calc(100% / 3 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [fourColumnResponseMediaQuery]: {
+    width: `calc(100% / 4 - ${ResponseCardDimensions.GAP}px)`
+  },
+  [fiveColumnResponseMediaQuery]: {
+    width: `calc(100% / 5 - ${ResponseCardDimensions.GAP}px)`
```
Comment (BartoszJarocki) [inline packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx:51]: as the `grid` we're using is really just a flexbox that wraps into multiple rows there's no better way of splitting it into responsive columns

**Comment by BartoszJarocki** [comment]
@jmtaber129 would you mind taking a look?

**Review by jmtaber129 — CHANGES_REQUESTED** [review]
Looks way better!  Just one thing about how we're calculating breakpoint widths

File: packages/client/components/TeamPrompt/TeamPromptGridDimensions.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const ResponseCardDimensions = {
+  MIN_CARD_HEIGHT: 100,
+  GAP: 32,
+  PREFERRED_RESPONSE_WIDTH: 360
+}
+
+export const ResponsesGridBreakpoints = {
+  TWO_RESPONSE_COLUMN:
+    ResponseCardDimensions.PREFERRED_RESPONSE_WIDTH * 2 + ResponseCardDimensions.GAP,
```
[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply packages/client/components/TeamPrompt/TeamPromptGridDimensions.ts]: very true! things got a bit too cryptic. I've changed the calculations and added some comments

File: packages/client/components/TeamPrompt/TeamPromptGridDimensions.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export const ResponseCardDimensions = {
+  MIN_CARD_HEIGHT: 100,
+  GAP: 32,
+  PREFERRED_RESPONSE_WIDTH: 360
```
[the remark this answers is not in the document]
Comment (BartoszJarocki) [reply packages/client/components/TeamPrompt/TeamPromptGridDimensions.ts]: that's a better name, fixed

**Comment by BartoszJarocki** [comment]
@jmtaber129 would you mind doing another round? thanks!

**Pull request merged**